### ISSUE TITLE
For Linux 

### ISSUE THREAD

**Issue by Keerthivasan472** [x-Ai/BurpSuite#19]
Is it available for Linux platform or we can modify the configuration to work with Linux

**Comment by x-Ai** [comment]
> Is it available for Linux platform or we can modify the configuration to work with Linux

making it work on Linux is also simple, you can insert the statement into the corresponding file yourself. I don't use burp on Linux, so I didn't explain how to run it in the README.md.

**Comment by Keerthivasan472** [comment]
Can u tell me which file it is for Linux , I can't find it .
Burpsuiteloader can't work in burpsuitepro 2023.9.3 version

**Comment by x-Ai** [comment]
> Can u tell me which file it is for Linux , I can't find it . Burpsuiteloader can't work in burpsuitepro 2023.9.3 version

you can insert it into the BurpSuitePro.vmoptions file.

**Comment by Keerthivasan472** [comment]
Yeah I think it is , but the loader file not work for recent version of burpsuitepro y

**Comment by x-Ai** [comment]
> Yeah I think it is , but the loader file not work for recent version of burpsuitepro y

i understand what you mean. you need to figure out how to get the license on your own.

**Comment by Gill1000** [comment]
@Keerthivasan472 bro did you make this work in linux?

**Comment by Keerthivasan472** [comment]
Yeah!
